setup-and-install/action.yml

```diff
@@ -45,14 +48,28 @@ runs:
     - name: Cache local Maven repository
       uses: actions/cache@v4
       id: cache
-      if: ${{ inputs.cache == 'true' }}
+      if: ${{ inputs.cache == 'true' && inputs.build-tool == 'Maven' }}
       with:
         path: ~/.m2/repository
         key: ${{ inputs.cache-name }}-${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
         restore-keys: |
           ${{ inputs.cache-name }}-${{ runner.os }}-maven-
 
-    - name: Update maven dependencies
+    - name: Update Maven dependencies
       working-directory: ${{ inputs.working-directory }}
+      if: ${{ inputs.build-tool == 'Maven' }}
       run: ./mvnw --batch-mode --fail-fast dependency:resolve dependency:resolve-plugins
       shell: bash
+
+    - name: Setup Gradle
+      uses: gradle/actions/setup-gradle@v4
+      if: ${{ inputs.build-tool == 'Gradle' }}
+      with:
+        cache-disabled: ${{ inputs.cache == 'false' }}
+        cache-read-only: 'false'
+
+    - name: Update project configuration and Gradle dependencies
```

Comment:
I'm just curios about the wording. Does here really get the project configuration changed?

Reply:
Ok, you are right. But is this an information that is relevant for the user that sees this step in the actions? Or could we just simply call it "Update dependencies" and that is what the user has to know?